hibike/hibike_process.py

```diff
@@ -0,0 +1,191 @@
+import time, random
+import threading
+import multiprocessing
+import hibike_message as hm
+import queue
+import glob
+import serial
+__all__ = ["hibike_process"]
+
+fake_uids = [0 << 72, 7 << 72]
+
+
+uid_to_index = {}
+
+def hibike_process(badThingsQueue, stateQueue, pipeFromChild):
+
+    ports = glob.glob("/dev/ttyACM*") + glob.glob("/dev/ttyUSB*") + open("virtual_devices.txt", "r").read().split()
```

Comment:
You should only use `virtual_devices.txt` if it is present. (Since that file should be gitignored, it may not be present all the time.)